here-naksha-lib-psql/src/test/java/com/here/naksha/lib/psql/PsqlTests.java

```diff
@@ -142,6 +147,26 @@ static void beforeTest() {
     NakshaContext.currentContext().setAppId("naksha-lib-psql-unit-tests");
     nakshaContext = new NakshaContext().withAppId(TEST_APP_ID).withAuthor(TEST_AUTHOR);
     fg = new PsqlFeatureGenerator();
+    final String url;
+    if (existingUrl != null) {
+      url = existingUrl;
+    } else {
+      final DockerImageName image = DockerImageName.parse("greenoag/postgres-plv8-postgis:15.2-3.1.5-3.3")
+                      .asCompatibleSubstituteFor("postgres");
+      postgreSQLContainer =
+              new PostgreSQLContainer<>(image)
+                      .withDatabaseName("unimap")
+                      .withUsername("postgres")
+                      .withPassword("postgres");
+
+      postgreSQLContainer.start();
```

Comment:
I think that when you manually control TestContainer's lifecycle then you also need to manually stop it once all dependent tests are done.
This page would suggest that: https://java.testcontainers.org/test_framework_integration/manual_lifecycle_control/

For automated lifecycle, you can use their annotations, see: https://java.testcontainers.org/test_framework_integration/junit_5/

Reply:
Probably we don't notice because the container is removed when the JVM shuts down, but I will manually stop it.

Reply:
Fixed.

Reply:
> the container is removed when the JVM shuts down

I was unaware of that, I thought that TestContainers only delegates commands to running docker engine and you can end up with stale container if your JVM goes down without stopping the container